build.sbt

```diff
@@ -19,4 +19,3 @@ lazy val djinni = (project in file("."))
     assembly / assemblyOption := (assembly / assemblyOption).value.copy(prependShellScript = Some(defaultUniversalScript(shebang = false))),
     assembly / test := {}
   )
-  inConfig(IntegrationTest)(org.scalafmt.sbt.ScalafmtPlugin.scalafmtConfigSettings)
```

Comment:
documentation says this is required
https://scalameta.org/scalafmt/docs/installation.html#enable-integrationtest
to also format the integration tests, and not only the source code

Reply:
It is not necessary, because i open an issue. Maybe the docs are not updated.
https://github.com/scalameta/sbt-scalafmt/issues/152

Reply:
I can confirm that. Thanks, @paulo-coutinho !

Reply:
indeed